persist/fs/seek_manager_test.go

```diff
@@ -21,14 +21,22 @@
 package fs
 
 import (
+	"sync"
 	"testing"
+	"time"
 
+	xtime "github.com/m3db/m3x/time"
+
+	"github.com/fortytw2/leaktest"
+	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/require"
 )
 
 func TestSeekerManagerCacheShardIndices(t *testing.T) {
+	defer leaktest.CheckTimeout(t, 1*time.Minute)()
```

Comment:
Any reason why 1minute here too?

Reply:
same reason as before, travis + race detector is sloooooow and timeout starts from beginning of test not the end

Reply:
Makes sense if from beginning of test, strange design choice there for the API... np.